app/src/pages/admin/serverSettingsPage/emailServerTab/emailServerTabFormFields/emailServerTabFormFields.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import classNames from 'classnames/bind';
+import { formValueSelector } from 'redux-form';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import { activeProjectSelector } from 'controllers/user';
+import { FormField } from 'components/fields/formField';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import { Input } from 'components/inputs/input';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { InputCheckbox } from 'components/inputs/inputCheckbox';
+import {
+  EMAIL_SERVER_FORM,
+  AUTH_ENABLED_KEY,
+  PROTOCOL_KEY,
+  SSL_KEY,
+  TLS_KEY,
+  FROM_KEY,
+  HOST_KEY,
+  PORT_KEY,
+  USERNAME_KEY,
+  PASSWORD_KEY,
+} from '../constants';
+import styles from './emailServerTabFormFields.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  hostLabel: {
+    id: 'EmailServerTabFormFields.hostLabel',
+    defaultMessage: 'Host',
+  },
+  protocolLabel: {
+    id: 'EmailServerTabFormFields.protocolLabel',
+    defaultMessage: 'Protocol',
+  },
+  fromLabel: {
+    id: 'EmailServerTabFormFields.fromLabel',
+    defaultMessage: 'Default sender name',
+  },
+  portLabel: {
+    id: 'EmailServerTabFormFields.portLabel',
+    defaultMessage: 'Port',
+  },
+  authLabel: {
+    id: 'EmailServerTabFormFields.authLabel',
+    defaultMessage: 'Authorization',
+  },
+  usernameLabel: {
+    id: 'EmailServerTabFormFields.usernameLabel',
+    defaultMessage: 'Username',
+  },
+  passwordLabel: {
+    id: 'EmailServerTabFormFields.passwordLabel',
+    defaultMessage: 'Password',
+  },
+});
+
+@connect((state) => ({
+  projectId: activeProjectSelector(state),
+  authEnabled: formValueSelector(EMAIL_SERVER_FORM)(state, AUTH_ENABLED_KEY),
+}))
+@injectIntl
+export class EmailServerTabFormFields extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    authEnabled: PropTypes.bool,
+    changeFieldValue: PropTypes.func,
+  };
+
+  static defaultProps = {
+    authEnabled: false,
+    changeFieldValue: () => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.protocolOptions = [{ value: 'smtp', label: 'SMTP' }];
+    this.authOptions = [{ value: true, label: 'ON' }, { value: false, label: 'OFF' }];
+  }
+
+  onChangeAuthAvailability = (arg, value) => {
```

Comment:
`event, value` - I think it looks more clear

Reply:
Fixed